llms/mlx_lm/models/stablelm_epoch.py

```diff
@@ -45,19 +44,20 @@ def __init__(self, config: ModelArgs):
                 f"hidden_size must be divisible by num_heads (got `hidden_size`: {self.hidden_size}"
                 f" and `num_heads`: {self.num_heads})."
             )
+        self.use_qkv_bias = True if config.vocab_size > 50304 else False
```

Comment:
Is this available in the config?

Reply:
I added the same key to the HuggingFace config.json so now we don't need to use the hard coded value.